pkgs/development/python-modules/nbconvert/default.nix

```diff
@@ -1,34 +1,37 @@
-{ lib
+{ beautifulsoup4
+, bleach
 , buildPythonPackage
-, fetchPypi
-, pytestCheckHook
-, glibcLocales
+, defusedxml
 , entrypoints
-, bleach
-, beautifulsoup4
-, mistune
-, nbclient
+, fetchPypi
+, ipykernel
+, ipywidgets
 , jinja2
-, pygments
-, traitlets
-, testpath
 , jupyter_core
+, jupyter-client
 , jupyterlab-pygments
+, lib
+, mistune
+, nbclient
 , nbformat
-, ipykernel
 , pandocfilters
+, pygments
+, pyppeteer
+, pytestCheckHook
+, testpath
+, tinycss2
 , tornado
-, jupyter-client
-, defusedxml
+, traitlets
 }:
 
 buildPythonPackage rec {
   pname = "nbconvert";
-  version = "6.4.5";
+  version = "6.5.0";
+  format = "setuptools";
```

Comment:
can we use pyproject?

Reply:
same as https://github.com/NixOS/nixpkgs/pull/171563#discussion_r865310873